### ISSUE TITLE
About translation effect

### ISSUE THREAD

**Issue by robertjyh** [woshisad159/TFNet#1]
Your dataset makes sign language tasks closer to real application scenarios. Thank you for your contribution!
In real environments, in addition to recognition tasks, we also need to face translation tasks. I used your dataset to train a translation model, but the effect was very poor. This may be because the dataset itself is difficult or my method is not reasonable enough.
Have you tested the translation effect on this dataset?
Thanks!

**Comment by woshisad159** [comment]
I have not tested the translation effect of this dataset, my research direction is only in the direction of sign language recognition, and actually the recognition effect is not good, the WER of the test set is only 41.9%. The spoken translation is given to describe clearly how this dataset is built in the process. I know a way to improve the recognition performance dramatically, you can improve the model performance dramatically by pre-training the model in CSL-Daily first, and then train it in CE-CSL dataset, the recognition WER of the test set can reach about 32%, I don't know if this method is effective for you.
Thanks!






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "woshisad159/TFNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月2日(星期六) 晚上7:33
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[woshisad159/TFNet] About translation effect (Issue #1)





 
Your dataset makes sign language tasks closer to real application scenarios. Thank you for your contribution!
 In real environments, in addition to recognition tasks, we also need to face translation tasks. I used your dataset to train a translation model, but the effect was very poor. This may be because the dataset itself is difficult or my method is not reasonable enough.
 Have you tested the translation effect on this dataset?
 Thanks!

**Comment by robertjyh** [comment]
Thank you for your supplement! I will try to use CSL-Daily to improve my training effect.

**Comment by Lee-xeo** [comment]
For translation problems, I recommend using a large Chinese language model for fine-tuning, MengZi-T5 performs well on this problem, and the training overhead is also very small (about 8G video memory)

**Comment by nanana-121** [comment]
> For translation problems, I recommend using a large Chinese language model for fine-tuning, MengZi-T5 performs well on this problem, and the training overhead is also very small (about 8G video memory)

Hello, have you tried fine-tuning the model? I found that its actual recognition performance is quite poor. I would appreciate any advice you could provide.

**Comment by woshisad159** [comment]
Thank you very much for sharing, I'll update a trained parameter in the README, hope it helps. Recognition accuracy can be greatly improved by pre-training the model on CSL-Daily first.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "woshisad159/TFNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年3月13日(星期四) 下午2:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [woshisad159/TFNet] About translation effect (Issue #1)



    
For translation problems, I recommend using a large Chinese language model for fine-tuning, MengZi-T5 performs well on this problem, and the training overhead is also very small (about 8G video memory)
  
Hello, have you tried fine-tuning the model? I found that its actual recognition performance is quite poor. I would appreciate any advice you could provide.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***&gt;
  nanana-121 left a comment (woshisad159/TFNet#1)
  
For translation problems, I recommend using a large Chinese language model for fine-tuning, MengZi-T5 performs well on this problem, and the training overhead is also very small (about 8G video memory)
  
Hello, have you tried fine-tuning the model? I found that its actual recognition performance is quite poor. I would appreciate any advice you could provide.
